pinc/new_user_mails.inc

```diff
@@ -161,6 +161,15 @@ and browse the resources in our Information for New and
 Returning Volunteers area at
 $dp_wiki_url/DP_Official_Documentation:General#Information_For_New_and_Returning_Volunteers
 
+
+Certification of Volunteer Work
+-------------------------------
+
+Distributed Proofreaders tracks only the number of pages proofread. It
```

Comment:
`Distributed Proofreaders` should be `$site_name`. Also should it be "We do not track" vs "It does not track"?

Reply:
Done, and thanks -- I feel kind of silly for missing `$site_name`. And "We" sounds like a good update.